examples/pytorch/rgcn/experimental/get_mag_data.py

```diff
@@ -48,12 +48,12 @@
         for name in hg.nodes[ntype].data:
             node_data[ntype + '/' + name] = hg.nodes[ntype].data[name][local_nodes]
     print('node features:', node_data.keys())
-    dgl.data.utils.save_tensors(metadata['part-{}'.format(part_id)]['node_feats'], node_data)
+    dgl.data.utils.save_tensors('outputs/' + metadata['part-{}'.format(part_id)]['node_feats'], node_data)
```

Comment:
It only modifies the saving path of `node_data` from current directory `.` to `outputs/`. How does this adapt the changes from absolute path to relative path?